examples/demo/configuration.nix

```diff
@@ -23,8 +23,8 @@ in
   ];
 
   disabledModules = [
-    <nixpkgs/nixos/modules/installer/cd-dvd/iso-image.nix>
-    <nixpkgs/nixos/modules/installer/cd-dvd/installation-cd-base.nix>
+    "installer/cd-dvd/iso-image.nix"
+    "installer/cd-dvd/installation-cd-base.nix"
```

Comment:
This is where it is unnecessary to have the full nixpkgs path, because we are pattern-matching by relative path. See https://nixos.org/manual/nixos/stable/#sec-replace-modules